package.json

```diff
@@ -79,6 +80,7 @@
     "eslint-plugin-jsx-a11y": "^4.0.0",
     "eslint-plugin-react": "^6.10.0",
     "jest": "^19.0.2",
+    "ncp": "^2.0.0",
```

Comment:
Is there no other way we can do this? I mean, copying the config file to the required path. If there is no other way, then I'll merge this.